alibi_detect/cd/pytorch/tests/test_lsdd_online_pt.py

```diff
@@ -22,12 +22,20 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.dense2(x)
 
 
+def preprocess_list(x: List[np.ndarray]) -> np.ndarray:
+    if len(x) > 1:
+        return np.concatenate(x, axis=0)
+    else:
+        return np.array(x)[0]
```

Comment:
I believe `np.concatenate` works even on lists of length 1 so no special case should be necessary.

Reply:
They are 2 different things, the first case is for the reference data, the second case is a test on the `Any` type for predict where `Any` is actually a list.